site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst

```diff
@@ -17,6 +17,7 @@ JavaScript and compiled C or C++:
   created with:
 
   - :ref:`Embind or WebIDL-Binder<interacting-with-code-binding-cpp>`
+  - :ref:`Emnapi (Node-API)<interacting-with-code-emnapi>`
```

Comment:
I'm not sure it makes sense to list it here.  Emnapi is specific set of bindings for node API, right?  Not a method of writing new binding?

Reply:
In my understanding, developers cannot use Node-API to write bindings in emscripten before emnapi exists, in a sense it is a new way to write bindings in emscripten, so it is listed here like embind.